### ISSUE TITLE
Suggestion: Needless blank line(s) additions

### ISSUE THREAD

**Issue by vanniktech** [shyiko/ktlint#65]
The check, Needless blank line(s), flags the following code:

```kotlin
}


private fun something() { }
```

although I'd also expect it to flag the one below as well:

```kotlin
  }

}
```

since the newline between the `}` is not necessary.

**Comment by shyiko** [comment]
Hi @vanniktech. 

While I like the idea, I'm note sure Intellij built-in formatter supports this (there is a setting for Java (Code Style -> Java -> Blank Lines -> Before '}') which can be set to 0 to enforce this kind of style, but not for Kotlin) (at least not in IDEA 2016.3). It's pretty easy to implement but I'd prefer not to add anything that can't be enforced by the IDEA's formatter (if we can).

**Comment by vanniktech** [comment]
[There will be an option with 1.1.4[(https://youtrack.jetbrains.com/issue/KT-15504).

**Comment by shyiko** [comment]
@vanniktech :+1:

**Comment by vanniktech** [comment]
Will this mean this functionality will be added now or once 1.1.4 is out?

**Comment by shyiko** [comment]
I think we should hold off until 1.1.4 is out (just to be on the safe side).

**Comment by vanniktech** [comment]
[Kotlin 1.1.4 is out](https://blog.jetbrains.com/kotlin/2017/08/kotlin-1-1-4-is-out/)

**Comment by shyiko** [comment]
Changed contributed by @Ilya-Gh :tada:. Keeping this issue open until `ktlint-intellij-idea-integration` is updated too.

**Comment by vanniktech** [comment]
This can be closed now. Thanks :)
